src/main/resources/db/changelog/changes/10-create-category-table.yaml

```
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-category-table
+      author: Olena
+      changes:
+        - createTable:
+            tableName: categories
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(255)
```

description can be long
```suggestion
                  type: text
```